tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -11,66 +11,74 @@ internal static class Program
     private static unsafe void Main(string[] args)
     {
         Add(1, 2);
-
+      
         int tempInt = 0;
         (*(&tempInt)) = 9;
-
-        TwoByteStr str = new TwoByteStr() { first = 1, second = 2 };
-        TwoByteStr str2 = new TwoByteStr() { first = 3, second = 4 }; ;
-        *(&str) = str2;
-        str2 = *(&str);
-
-        if (tempInt == 9)
+	
+        if(tempInt == 9)
         {
-            string s = "Hello from C#!";
-            PrintString(s);
+			PrintLine("Hello from C#!");
```

Comment:
Nit -- indentation. Also, is this a tab? If so, please use spaces (and if I missed other tabs in this, please fix them as well)

Reply:
Yes, I got this wrong in the merge.